modules/standard/BigInteger.chpl

```diff
@@ -2922,27 +2944,52 @@ module BigInteger {
     return ret.safeCast(uint);
   }
 
-  pragma "last resort"
-  @deprecated(notes="The 'starting_bit' argument is deprecated, please use 'startBitIdx' instead")
-  proc bigint.scan0(starting_bit: integral) : uint {
-    return this.scan0(startBitIdx = starting_bit);
-  }
+  /*Scan ``this``, starting from ``startBitIdx``, towards more significant
```

Comment:
I think this is missing a newline after the `/*`, but I'm not sure that affects documentation?

Reply:
It doesn't affect the documentation, but I can appreciate from a stylistic point of view it is cleaner. Fixing